commands/xp.py

```diff
@@ -4,7 +4,7 @@
 import utils.colours as colour
 import utils.values as value
 import utils.database as sql
-from utils.helpers import discordError
+from utils.helpers import discordError, cinfo, add_exp

```

Comment:
Please don't do this, we import the whole of helpers for a reason. Also, cinfo does not exist. Please just import utils.helper and not worry about what methods to include.